roles/pulibrary.figgy/tasks/main.yml

```diff
@@ -155,3 +155,11 @@
   shell: curl -L {{ simple_tiles_url }} | tar -xz && cd simple-tiles-{{simple_tiles_version}} && ./configure && make && make install
   args:
     creates: /usr/local/lib/libsimple-tiles.so
+- name: Install subversion client
+  apt:
+    name: subversion
+    state: present
+- name: Configure subversion to not save passwords
```

Comment:
This fails for me in the Vagrant Box with the following:
```
Destination directory /home/deploy/.subversion does not exist
```

Reply:
@jrgriffiniii thank you for catching that! I changed it to go in global config instead. I also moved this to a role since it was easier to test that way.